ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -5623,4 +5616,16 @@ RFVSC2X:
   - return_fan_run_status_2
   - return_fan_speed_percentage_command_2
   implements:
-  - OPERATIONAL
\ No newline at end of file
+  - OPERATIONAL
+
+ALM:
```

Comment:
Be specific, too broad. What kind of alarms are these? And do they perhaps belong as optional fields on other abstract types (e.g. low discharge air flowrate alarm might belong on DFSS since it will be caused by fan failure).

Reply:
I see sir. I'll adjust this and limit to general alarms, which would be:

opt_uses:
- lost_power_alarm
- failed_alarm

but not:

- fire_alarm
- water_leak_detection_alarm

or do I remove the whole ALM perhaps?